src/DependencyInjection/MonologExtension.php

```diff
@@ -292,6 +297,44 @@ private function buildHandler(ContainerBuilder $container, $name, array $handler
                 ]);
                 break;
 
+            case 'mongodb':
+                if (!class_exists('MongoDB\Client')) {
+                    throw new \RuntimeException('The "mongodb" handler requires the mongodb/mongodb package to be installed.');
+                }
+
+                if (isset($handler['mongodb']['id'])) {
+                    $client = new Reference($handler['mongodb']['id']);
+                } else {
+                    $uriOptions = ['appname' => 'monolog-bundle'];
+
+                    if (isset($handler['mongodb']['username'])) {
+                        $uriOptions['username'] = $handler['mongodb']['username'];
+                    }
+
+                    if (isset($handler['mongodb']['password'])) {
+                        $uriOptions['password'] = $handler['mongodb']['password'];
+                    }
+
+                    $client = new Definition('MongoDB\Client', [
+                        $handler['mongodb']['uri'],
+                        $uriOptions,
```

Comment:
Should we have `writeConcern: 0 or 1` by default for logs?
You can accept driver options.